src/main/kotlin/me/spradling/gift/core/api/RestVerticle.kt

```diff
@@ -0,0 +1,30 @@
+package me.spradling.gift.core.api
+
+import io.vertx.core.AbstractVerticle
+import io.vertx.core.Future
+import io.vertx.ext.web.Router
+import me.spradling.gift.core.api.routes.HealthHandler
+import javax.inject.Inject
+
+class RestVerticle @Inject constructor(): AbstractVerticle() {
+
+  override fun start(startFuture : Future<Void>) {
+    val start = Future.future<Void>()
+
+    val router = Router.router(vertx)
+
+    router.get("/health").handler(HealthHandler())
+
+    vertx.createHttpServer()
+        .requestHandler(router::accept)
+        .listen(8080) { result ->
```

Comment:
Do you want to make this a constant or just roll with it hardcoded here?

Reply:
This will be address when the configuration is created!